src/Model/Page.php

```diff
@@ -308,6 +308,11 @@ public function getName()
      */
     public function setSlug($slug): void
     {
+        if ($slug === null) {
+            $this->slug = null;
+            return;
+        }
+        
         $this->slug = self::slugify(trim($slug));
```

Comment:
isn't it better to simply do: `self::slugify(trim((string) $slug));`

Reply:
This depends on whether you want to store NULL or an empty string into the object.

You decide! :)

Reply:
https://3v4l.org/i2Ito

Seems like cast to string preserves better the old code